documentation/docs/scripts/fileUtils.md

```diff
@@ -51,11 +52,12 @@ none
 none
 
 #### Exceptions
-* `IllegalArgumentException`: If the parameter `dir` is null or empty.
-* `AbortException`: If the directory does not exist or is not a directory or the directory is empty.
+* `IllegalArgumentException`
```

Comment:
I think that `-` instead of `:` would be enough.

Reply:
I agree if there is one exception. What about cases where there is more than one exception, as is the case in mtaBuild:

## Exceptions
* `AbortException`
    * If there is an invalid `buildTarget`.
    * If there is no key `ID` inside the `mta.yaml` file.

Reply:
I agree if there is one exception. What about cases where there is more than one exception, as is the case in mtaBuild:

## Exceptions
* `AbortException`
    * If there is an invalid `buildTarget`.
    * If there is no key `ID` inside the `mta.yaml` file.

Reply:
Should we have two cases: One for the single exception and two for multiple exceptions: <br>
1. Single exception:
* `Exception` - If `archivePath` is not provided.
<br>

2. Multiple exceptions:
* `AbortException`:
    * If neo-java-web-sdk is not installed, or `neoHome`is wrong.
    * If `deployHost` is wrong.
    * If `deployAccount` is wrong.

Reply:
I would inline multiple exceptions:

* `AbortException`- If neo-java-web-sdk is not installed, or if `neoHome`, `deployHost`, or deployAccount` is wrong.

But it is just a suggestion.